package/CHANGELOG

```diff
@@ -13,7 +13,7 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/?? IAlibay, melomcr, mdd31, ianmkenney, richardjgowers, hmacdope
+??/??/?? IAlibay, melomcr, mdd31, ianmkenney, richardjgowers, hmacdope, ojeda-e
```

Comment:
This is going to have to spillover to the next line next time someone adds an entry - we should just break it into two lines now to save time later.